src/book/03-guides/04-general/01-learning-resources.mdx

```diff
@@ -0,0 +1,180 @@
+---
+section: Guides
+chapter: General
+title: Learning Resources
+description: Where to find out how to learn about the various aspects of NUbots.
+slug: /guides/general/learning-resources
+---
+
+This page aims to provide direction and resources on where to go to learn more about various areas of NUbots. There is a mixture of resources available, including external resources and material prepared by NUbots team members. The main resources are Reading pages, videos and workshops. Workshops are run in lab, or on Zoom when not possible to do in the lab, by a member of the team. LinkedIn Learning has many great videos, some of which are linked here - as a UoN student, you can get access to LinkedIn Learning for free.
+
+## General
+
+### Git
+
+The NUbots team uses Git and GitHub in its development workflow. Almost any project done on the NUbots team will involve using Git and GitHub.
+
+**Reading:**
+
+- [NUbook: Git and PR Workflow](/guides/general/git)
+
+**Videos:**
+
+- LinkedIn Learning: Git Concepts
+- LinkedIn Learning: Git Command Line
+- LinkedIn Learning: GitKraken
+- LinkedIn Learning: GitHub Desktop
+
+**Tutorials:**
```

Comment:
I'm not sure on the heading. Stuff in here includes NUbots workshops, online courses and online tutorials.

Reply:
I like tutorials